react-basic-practice/src/App.tsx

```diff
@@ -1,17 +1,15 @@
 import React from "react";
 import { ChakraProvider } from "@chakra-ui/react";
-import theme from "./themes";
-import Search from "@components/Search";
+import { theme } from "themes";
 import Button from "@components/Button";
 const App: React.FC = () => {
   return (
     <ChakraProvider theme={theme}>
       <div style={{padding:"20px 50px"}}>
```

Comment:
Please use Box component

Reply:
I use it just test, i'll update comment line after